code/modules/clothing/rogueclothes/neck.dm

```diff
@@ -37,6 +37,70 @@
 	toggle_icon_state = TRUE
 	sewrepair = TRUE
 
+/obj/item/clothing/neck/roguetown/coif/padded
+	name = "padded coif"
+	desc = "Cheap and easy to make. It's better than leaving your head exposed."
+	icon_state = "ccoif"
+	item_state = "ccoif"
+	color = null
+	flags_inv = HIDEHAIR
+	slot_flags = ITEM_SLOT_NECK|ITEM_SLOT_HEAD
+	blocksound = SOFTHIT
+	body_parts_covered = NECK|HAIR|EARS|HEAD
+	armor = ARMOR_PADDED //gambeson for head
+	prevent_crits = list(BCLASS_CUT, BCLASS_BLUNT)
+	adjustable = CAN_CADJUST
+	toggle_icon_state = TRUE
+	sewrepair = TRUE
+
+/obj/item/clothing/neck/roguetown/coif/heavypadding
+	name = "heavy padded coif"
+	desc = "Cheap and easy to make, used on poor soldiers this coif keeps your face warm and covered, pair with a helmet."
```

Comment:
```suggestion
	desc = "A heavier padded coif meant to be worn on its own or under a helmet. Layered properly, it can last through even the busiest of daes."
```